docs/materials/htcondor/part1-ex1-login.md

```diff
@@ -94,7 +94,7 @@ As you can see from the output, we are using HTCondor 9.11.0.
 
 ### FYI: Background information about HTCondor version numbers
 
-HTCondor always has two types of releases at one time: long-term-support (LTS) releases and feature releases. HTCondor 9.0.x is considered an LTS, indicated by a 0 in the second digit. Within one stable series, all versions have the same features (for example 9.0.5 and 9.0.10 have the same set of features) and differ only in bug and security fixes.
+HTCondor always has two types of releases at one time: long-term-support (LTS) releases and feature releases. HTCondor 9.0.x is considered a LTS release, indicated by a 0 in the second digit. Within one LTS series, all versions have the same features (for example 9.0.5 and 9.0.10 have the same set of features) and differ only in bug and security fixes.
```

Comment:
I would still go with “an” before “LTS”, so “HTCondor 9.0.x is considered an LTS release, …” For me, at least, that matches how we would actually say it out loud.